thirdparty/misc/r128.h

```diff
@@ -1643,7 +1643,7 @@ void r128Shl(R128 *dst, const R128 *src, int amount)
       r[1] = r[0] << (amount - 64);
       r[0] = 0;
    } else if (amount) {
-#  ifdef _M_X64
+#  if defined(_M_X64) && !defined(R128_STDC_ONLY)
```

Comment:
Can we make sure this gets to mbedtls upstream or at least a diff patch is generated?

Reply:
https://github.com/fahickman/r128/pull/15